keyboards/gboards/k/butterstick/config.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1337
```

Comment:
This Vendor ID is assigned to Radiant Networks Plc.

https://www.the-sz.com/products/usbid/index.php?v=0x1337

Please change it.

Reply:
Changed in https://github.com/qmk/qmk_firmware/pull/8922/commits/bb85f4cbe79eee1f8869105959890ee3f92906e7